web/server/codechecker_server/api/report_server.py

```diff
@@ -349,6 +347,81 @@ def process_report_filter(session, report_filter):
     return filter_expr
 
 
+def get_outstanding_reports_filter(tbl=Report):
```

Comment:
this was renamed to open reports right?

what is the `tbl=Report` expression, Report is a type right?